### ISSUE TITLE
特别好的主题，想问个问题

### ISSUE THREAD

**Issue by Lanjidechao** [MichaelLaw2003/amber-vertical-typora-theme#3]
虽然有点倒反天罡但是想问一下作者，有没有横板左向右的魔改方法……
想用这个主题写小说，竖版确实有点不熟悉

**Comment by MichaelLaw2003** [comment]
Yes, of course. Here you are. Just copy the CSS file below to your theme directory.
[amber-for-lanjidechao.zip](https://github.com/user-attachments/files/16925083/amber-for-lanjidechao.zip)

**Comment by MichaelLaw2003** [comment]
The fonts are a bit problematic in a horizontal layout. But I don't really know much about font. 
I'll work on that, but maybe I'll have to contact the author of the fonts.
